packages/tsx-editor/README.md

```diff
@@ -1,14 +1,64 @@
 # @uifabric/tsx-editor
 
-Monaco-based TypeScript+React live editor used for examples on the Fabric website.
+Monaco-based TypeScript+React live code editor with full typing support. It was primarily written for component examples on the [Fabric website](https://developer.microsoft.com/en-us/fabric#/controls/web), but it can be configured to work with other libraries too.
 
-The editor component's API is **highly unstable**, so it **should not** be used outside the Fabric repo yet.
+**WARNING:** The editor component's API is still **highly unstable**, so it **should not** be used outside the Fabric repo yet.
 
-## Required editor configuration
+## Features
 
-Any project consuming `@uifabric/tsx-editor` should follow the Webpack and runtime configuration instructions from the [`@uifabric/monaco-editor` readme](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/monaco-editor/README.md).
+### Live updating as user types
 
-Note that the helpers used arere-exported from this package for convenience:
+As the user edits TypeScript+React example code, it will be transpiled and rendered onto the page.
```

Comment:
```suggestion
As the user edits TypeScript and React example code, it will be transpiled and rendered onto the page.
```

Reply:
I don't really like either "TypeScript+React" or "TypeScript and React" here, but I'm not sure what other concise way to say it...